<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCheckTapeIncldHanWidget</class>
 <widget class="QWidget" name="CCheckTapeIncldHanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>8</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vlay_widget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlay_pam">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>位置偏移</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>1Y偏移</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spin_YBia1">
                  <property name="minimum">
                   <number>-999</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>-200</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>2Y偏移</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>3Y偏移</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>4Y偏移</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spin_YBia2">
                  <property name="minimum">
                   <number>-999</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>-200</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spin_YBia3">
                  <property name="minimum">
                   <number>-999</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>200</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSpinBox" name="spin_YBia4">
                  <property name="minimum">
                   <number>-999</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>200</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_changePos">
                <property name="text">
                 <string>调试</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>检测参数</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>阈值下限</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spin_minThre">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>阈值上限</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spin_maxThre">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>240</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_pamRead">
             <property name="text">
              <string>参数读取</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_detect">
             <property name="text">
              <string>检测</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_YBia1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_changePos</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>827</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_YBia2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_changePos</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_YBia3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_changePos</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_YBia4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_changePos</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>838</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>832</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
